host_reference/hisq_force_reference.cpp

```diff
@@ -4,10 +4,10 @@
 #include <string.h>
 #include <type_traits>
 
-#include "quda.h"
-#include "test_util.h"
-#include "misc.h"
-#include "hisq_force_reference.h"
+#include <quda.h>
```

Comment:
Do we actually have a convention when we use <> and when "" for includes? @maddyscientist ?

Reply:
No convention that I've ever used: I always just used `< >` since it looked it better to my eye, but then you told me the standard convention was to use `" "` within a library and `< >` for externals.

Reply:
never mind. Sure we can clean that if we ever want to.